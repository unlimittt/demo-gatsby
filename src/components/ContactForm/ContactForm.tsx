import React from "react"
import styles from "./ContactForm.module.scss"

const ContactForm = () => (
  <form
    method="post"
    netlify-honeypot="bot-field"
    data-netlify="true"
    name="contact"
  >
    <input type="hidden" name="bot-field" />
    <input type="hidden" name="form-name" value="contact" />
    <p>
      <label>
        Your Name: <input type="text" name="name" />
      </label>
    </p>
    <p>
      <label>
        Your Email: <input type="email" name="email" />
      </label>
    </p>
    <p>
      <label>
        Message: <textarea name="message" />
      </label>
    </p>
    <p>
      <button type="submit">Send</button>
    </p>
  </form>
)

export default ContactForm
