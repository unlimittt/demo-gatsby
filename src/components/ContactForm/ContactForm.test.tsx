import React from "react"
import Enzyme, { shallow } from "enzyme"
import Adapter from "enzyme-adapter-react-16"
import ContactForm from "./ContactForm"

Enzyme.configure({ adapter: new Adapter() })

describe("ContactForm", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<ContactForm />)
    expect(wrapper).toMatchSnapshot()
  })
})
