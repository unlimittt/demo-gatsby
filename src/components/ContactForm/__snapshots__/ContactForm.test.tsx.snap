// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactForm renders correctly 1`] = `
<form
  data-netlify="true"
  method="post"
  name="contact"
  netlify-honeypot="bot-field"
>
  <input
    name="bot-field"
    type="hidden"
  />
  <input
    name="form-name"
    type="hidden"
    value="contact"
  />
  <p>
    <label>
      Your Name: 
      <input
        name="name"
        type="text"
      />
    </label>
  </p>
  <p>
    <label>
      Your Email: 
      <input
        name="email"
        type="email"
      />
    </label>
  </p>
  <p>
    <label>
      Message: 
      <textarea
        name="message"
      />
    </label>
  </p>
  <p>
    <button
      type="submit"
    >
      Send
    </button>
  </p>
</form>
`;
